### PR TITLE
upgrade frictionless to v5.13; fix tests

### PR DESCRIPTION
**Pull request by kbubnicki**

### CODE REVIEW

File: tests/helpers.py [hunk 1]
```diff
@@ -2,22 +2,28 @@
...
 from typing import Optional, List
-from frictionless import validate_package
+from frictionless import validate
+
 
 THIS_SCRIPT_PATH = Path(__file__).parent
 REPOSITORY_ROOT_PATH = THIS_SCRIPT_PATH / ".."
 EXAMPLE_PATH = REPOSITORY_ROOT_PATH / "example" / "datapackage.json"
 PROFILE_PATH = REPOSITORY_ROOT_PATH / "camtrap-dp-profile.json"
-RELAXED_PROFILE_PATH = REPOSITORY_ROOT_PATH / "camtrap-dp-profile-relaxed.json" # File created in later step
```
Comment (peterdesmet) [inline tests/helpers.py:11]: I would leave this as is (single line) to keep it consistent with the variables above.

File: tests/helpers.py [hunk 2]
```diff
@@ -2,22 +2,28 @@
...
+RELAXED_PROFILE_PATH = (
+    REPOSITORY_ROOT_PATH / "camtrap-dp-profile-relaxed.json"
+)  # File created in later step
 TABLE_SCHEMA_PATHS = [
     REPOSITORY_ROOT_PATH / "deployments-table-schema.json",
     REPOSITORY_ROOT_PATH / "media-table-schema.json",
     REPOSITORY_ROOT_PATH / "media-observations-table-schema.json",
-    REPOSITORY_ROOT_PATH / "event-observations-table-schema.json"
+    REPOSITORY_ROOT_PATH / "event-observations-table-schema.json",
```
Comment (peterdesmet) [inline tests/helpers.py:19]: Is it a convention to end arrays with `,`?

File: tests/helpers.py [hunk 3]
```diff
@@ -2,22 +2,28 @@
...
 ]
 
-def validate_package_print_and_exit(descriptor_data: dict, paths_to_delete: Optional[List] = None) -> None:
-    report = validate_package(descriptor_data)
+
+def validate_package_print_and_exit(
```
Comment (peterdesmet) [inline tests/helpers.py:23]: Is it a convention to move this to multiple lines? Same for the rest of the changes of this file.

File: tests/requirements.txt [hunk 4]
```diff
@@ -1,21 +1,28 @@
...
 charset-normalizer==2.0.7
 click==8.0.3
 colorama==0.4.4
 decorator==5.1.0
-frictionless==4.20.2
+frictionless==5.13.1
+humanize==4.6.0
 idna==3.3
 isodate==0.6.0
+Jinja2==3.1.2
```
Comment (peterdesmet) [inline tests/requirements.txt:13]: Are all these extra requirements dependencies of Frictionless v5?

**Review by peterdesmet — CHANGES_REQUESTED** [review]
Thanks! Tested locally and it works. The PR contains a number of style changes I would not include to keep it consistent with the rest of the code. See my separate comments.

File: tests/helpers.py (hunk 1 above)
Comment (peterdesmet) [reply tests/helpers.py:11]: I understand this is done automatically by "black" formatting because the line is too long.

**Comment by kbubnicki** [comment]
Thanks! I have used https://github.com/psf/black to automatically format all `.py` files. I would also suggest to keep this style of formatting for future work.

File: tests/requirements.txt (hunk 4 above)
Comment (kbubnicki) [reply tests/requirements.txt:13]: I have created venv using `requirements.txt` from the main branch and simply upgraded `frictionless` package to the newest version and this is the list I have got.

**Review by peterdesmet — APPROVED** [review]

**Pull request merged**